docusaurus.config.ts

```diff
@@ -13,7 +13,7 @@ const deployingToNetlify = process.env['DEPLOYING_TO_NETLIFY'] === 'TRUE' || fal
 
 const config: Config = {
   title: 'Validator Documentation',
-  tagline: 'Validate all the things',
+  tagline: 'Continuous validation for environments',
```

Comment:
"Validate any environment" ?

"Ad hoc and continuous validation for any environment" ?